datadog/resource_datadog_integration_aws_log_collection_test.go

```diff
@@ -19,17 +19,22 @@ resource "datadog_integration_aws" "account" {
   role_name                        = "testacc-datadog-integration-role"
 }
 
+resource "datadog_integration_aws_lambda_arn" "lambda" {
+  account_id = "%s"
```

Comment:
nit: could we use the variable from datadog_integration_aws.account.account_id instead of passing another `uniq`?
Same for the one below. I think in that case we'd only have to pass one or two `uniq` here

Reply:
Good call! Thanks